ETL/ETL/_handle.h

```diff
@@ -627,7 +664,274 @@ class rhandle : public handle<T>
 	*/
 }; // END of template class rhandle
 
+template <class T>
```

Comment:
Please move `rhandle_old` before `rhandle_new`. In this case `git` will show the changes better.